main.go

```diff
@@ -40,20 +40,17 @@ import (
 )
 
 const (
-	namespaceEnvVar    = "NAMESPACE"
-	configSecretEnvVar = "CONFIG"
+	namespaceEnvVarName         = "NAMESPACE"
+	addonParamsSecretEnvVarName = "ADDON_PARAMS_SECRET"
```

Comment:
I was under the impression we agreed to have an ADDON_NAME env var and not ADDON_PARAMS_SECRET env var as it will also benefit the other add-on features we are working on